engine/gamesys/src/gamesys/scripts/script_physics.cpp

```diff
@@ -1191,8 +1193,15 @@ namespace dmGameSystem
         void* comp_world = 0x0;
         GetCollisionObject(L, 1, collection, &comp, &comp_world);
         
-        dmhash_t group_hash = dmGameSystem::GetCollisionGroup(comp_world, comp);
-        dmScript::PushHash(L, group_hash);
+        dmhash_t group_hash;
+        dmGameSystem::GetCollisionGroup(comp_world, comp, group_hash);
```

Comment:
I though we came to the conclusion that I was wrong to comment on this (regarding this function lacking error checking)?
Can the GetCollisionGroup() really fail?

Reply:
Well, in the unlikely case that the user cleared the `co` group it will return a zero-hash. I wouldn't call that "really failing".